scripts/image-build.sh

```diff
@@ -0,0 +1,32 @@
+
+#!/bin/bash
+
+# This script builds the candid docker image and pushes it to the
+# configured docker registry,
+# Required environment variables:
+# - DOCKER_REGISTRY
```

Comment:
Should we push these as arguments to the script? Prevents us requiring env vars then